pom.xml

```diff
@@ -43,6 +44,27 @@
       <artifactId>mockito-core</artifactId>
       <version>2.3.4</version>
     </dependency>
+    <!-- PowerMock TestNG Module -->
+	<dependency>
+	  <groupId>org.powermock</groupId>
+	  <artifactId>powermock-module-testng</artifactId>
+	  <version>${powermock.version}</version>
+	  <scope>test</scope>
+	</dependency>
+	<!-- PowerMock JUnit 4.4+ Module -->
+	<dependency>
+	  <groupId>org.powermock</groupId>
+	  <artifactId>powermock-module-junit4</artifactId>
+	  <version>${powermock.version}</version>
+	  <scope>test</scope>
+	</dependency>
+	<!-- PowerMock Mockito2 API -->
+	<dependency>
+	  <groupId>org.powermock</groupId>
+	  <artifactId>powermock-api-mockito2</artifactId>
+	  <version>${powermock.version}</version>
+	  <scope>test</scope>
+	</dependency>
```

Comment:
Powermock didn't seem to be used?

Reply:
It was implemented when I had tried to make a mock of a static method. I am still not sure if it will be 100% necessary or not, but this will most likely be deleted if I determine it unnessecary.